### ISSUE TITLE
Picking a room when creating a package takes very long

### ISSUE THREAD

**Issue by nadjaweisskopf** [leihs/leihs#511]
Please analyse and think about a possible solution. 

First solution: On first letter/number the list is shown, not before. --> for edit page on react this solution could be a possible solution. 

We might need to see if the package page needs to be reactified

**Comment by phiku** [comment]
Reproduce:
1. New model form (e.g. https://test.leihs.zhdk.ch/manage/8bd16d45-056d-5590-bc7f-12849f034351/models/new)
2. Click add package.
3. Select "Toni-Areal" as building.
4. Try to select a room => very slow.

This seems to be still the old version of the item edit form. This is not done in React yet, so it was most likely always like this.

**Comment by phiku** [comment]
@nadjaweisskopf For me its unclear why on the create model formular one can add packages, but not items. So perhaps we just should remove the add package button.

**Comment by nadjaweisskopf** [comment]
@phiku the problem is not only in packages but also when editing existing items.

**Comment by phiku** [comment]
@nadjaweisskopf I cannot reproduce this. We will have to see in planning.

**Comment by eins78** [comment]
@nadjaweisskopf I also could not reproduce it in the case of "edit items". For "new model" it is still pretty bad, though.
Before we can decide how to solve this, can you please verify once more that it is not hanging for "edit items"?

I tried the following steps:
- list inventory
- open model line, click edit item
- change to "unknown building/- room", save
- go back, enter "Toni-Areal" as building
- after loading the rooms for ~1sec, rooms can be selected from list
- save

I did it for an normal item as well as a package, no browser hanging.

**Comment by nadjaweisskopf** [comment]
@eins78 ok. I'll visit Roger on Monday to reproduce the edit item behaviour.

**Comment by eins78** [comment]
@nadjaweisskopf Cool. *If* it's still a problem, call me and I will join you immediatly ;)

**Comment by nadjaweisskopf** [comment]
@eins78 Roger will create the film on monday. I will create another ticket for that bug so we can close this one.

**Comment by eins78** [comment]
solution: only fix for 'edit items'

**Comment by phiku** [comment]
@nadjaweisskopf Is fixed for create/edit item, but not yet for create package. Create packages are not yet implemented in react. The question is, does this happen often, that somebody is creating a package? We did not change anything there and the problem occured recently, so it seems before nobody has ever realized. Or the other explanation is, they used an older computer now. Its only unusable on slow computers, so perhaps the did it on faster computers before?

**Comment by nadjaweisskopf** [comment]
first create model and create package in react

**Comment by phiku** [comment]
@nadjaweisskopf Should be no problem anymore because of new packages:
http://phiku.ruby.zhdk.ch/manage/8bd16d45-056d-5590-bc7f-12849f034351/inventory

**Comment by nadjaweisskopf** [comment]
ticket closed, will be tested with seperate ticket
